content/docs/SUSHI/project/_index.md

```diff
@@ -25,18 +25,15 @@ The **sushi-config.yaml** file provides project configuration data to SUSHI. It
 
 ## Initializing a SUSHI Project
 
-Setting up a fully-featured FSH project can be complex, so SUSHI provides an `--init` argument to do it automatically. When `sushi --init` is run, SUSHI will request project information from the user:
-
-{{% small-pageinfo color="primary" %}}
-<span class="tag">SUSHI 3.0</span>For SUSHI 3.0.0 and later, use `sushi init`.
-{{% /small-pageinfo %}}
+Setting up a fully-featured FSH project can be complex, so SUSHI provides an `init` command to do it automatically. When `sushi init` is run, SUSHI will request project information from the user:
```

Comment:
I didn't realize that we not only _included_ the SUSHI 2.x information, but we actually *preferred* the 2.x commands and treated the 3.x ones like the oddballs! Yikes!

Reply:
Yeah, we were really trying to ease people into the changes. But that's over now.